server/app/services/applicant/question/Question.java

```diff
@@ -100,11 +102,41 @@ && getAllPaths().stream().anyMatch(failedUpdates::containsKey)) {
 
   /**
    * A question is considered answered if the applicant data has been set for any of the paths
-   * associated with the question. If the applicant data does not contain the question's path, then
-   * it will be considered unanswered.
+   * associated with the question or if the question is tagged with a Primary Applicant Info Tag and
+   * the applicant has data saved in the corresponding column.
    */
   public boolean isAnswered() {
-    return getAllPaths().stream().anyMatch(p -> applicantQuestion.getApplicantData().hasPath(p));
+    boolean isAnsweredWithApplicantData =
+        getAllPaths().stream().anyMatch(applicantQuestion.getApplicantData()::hasPath);
+
+    // Filter to see if the applicant has data saved for any of the PAI tags on the question
+    // definition.
+    // If the returned set is NOT empty, then the applicant has data saved and the question is
+    // "answered"
+    ImmutableSet<PrimaryApplicantInfoTag> tags =
+        applicantQuestion.getQuestionDefinition().getPrimaryApplicantInfoTags();
+    boolean isAnsweredWithPrimaryApplicantInfo =
```

Comment:
This allows the answer to be shown on the preview screen.

Reply:
@nb1701 do you remember if we actually made a decision about this at some point or should i confirm this with product?

Reply:
That makes sense, I'll play around with this.

Reply:
done.